.github/workflows/ci.yml

```
@@ -0,0 +1,36 @@
+name: CI
+
+on:
+  push:        # 모든 브랜치 push 시 실행
+    branches: ["**"]
+  pull_request: # 모든 브랜치에서 PR 생성/업데이트 시 실행
+    branches: ["**"]
+  workflow_dispatch: {} 
```

현재 해당 브랜치에 push를 하면 pr로써, push로써 액션이 두 번 돕니다.
```suggestion
  push:        # main 브랜치 push 시 실행
    branches: [main]
  pull_request: # main 브랜치에서 PR 생성/업데이트 시 실행
    branches: [main]
  workflow_dispatch: {} 
```

---

해당 코드로 변경하여 액션 한 번 돌도록 수정했습니다.